<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigurationDialog</class>
 <widget class="QDialog" name="ConfigurationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="conf_scr">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="confcont_wid">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>584</width>
        <height>503</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="dim_grp">
         <property name="title">
          <string>Dimensions</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3"/>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="linefill_grp">
         <property name="title">
          <string>Lines and Fill style</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5"/>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="font_grp">
         <property name="title">
          <string>Font and Font Size</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4"/>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="output_grp">
         <property name="title">
          <string>Output</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6"/>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="input_grp">
         <property name="title">
          <string>Input</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7"/>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="histogram_grp">
         <property name="title">
          <string>Histogram</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8"/>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="data_grp">
         <property name="title">
          <string>Data rows</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_9"/>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="options_box">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Ok|QDialogButtonBox::Open|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
